### PR TITLE
Add Ruby 3.0 support, and add it to CI

### PR DESCRIPTION
**Pull request by taichi-ishitani**
This PR is to add Ruby 3.0 to CI.

### CODE REVIEW

**Comment by 64kramsystem** [comment]
Hello!

Thanks for the PR! I have bad news, though - Travis CI are now suspended, as Travis doesn't automatically grant free builds to open source projects.

I've asked information about applying, however, it's not straightforward (in particular, I have several open source projects, and application is per-project).

If you happened to have the interest and the time to convert the build to GitHub actions, that would be super-helpful, and immediately accepted :grimacing: :laughing: :+1:!

**Comment by taichi-ishitani** [comment]
Hi @saveriomiroddi ,

> GitHub actions

Sure. I will create the build env on Github actions.

**Comment by 64kramsystem** [comment]
Let me think for a few days, and I'll get back. I've just tried the build on 3.0 locally, and I've seen that it requires `rexml` manual import (and I've noticed your commit).

**Comment by taichi-ishitani** [comment]
I invoke `rake spec` on ruby 3.0 and I see no errors.

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 
 gemspec
+
+if RUBY_VERSION >= '3.0.0'
```
Comment (64kramsystem) [inline Gemfile]: While this code is working, the rigorous way of comparing versions is by using [`Gem::Version`](https://ruby-doc.org/stdlib-2.7.0/libdoc/rubygems/rdoc/Gem/Version.html).

Check this out:

```ruby
2.7.2 :001 > '3.0' > '10.0'
 => true 
2.7.2 :002 > Gem::Version.new('3.0') > Gem::Version.new('10.0')
 => false
```

:grin: 

When using the Ruby version, there's a ready API:

```ruby
2.7.2 :003 > Gem.ruby_version > Gem::Version.new('10.0')
 => false
```

Although the bad behavior will likely never be triggered, I place much value on exact semantics/logic :smile:.

**Comment by 64kramsystem** [comment]
So! If you change the commit with the more precise version comparison, I will merge this and the other PR!

Thanks for the contributions! :partying_face: :+1:

File: Gemfile (hunk 1 above)
Comment (taichi-ishitani) [reply Gemfile]: Thank you for reviewing!
I changed way to check Ruby version according to your proposal.

**Review by 64kramsystem — APPROVED** [review]

**Pull request merged**